src/jdk.jpackage/share/native/common/FileUtils.h

```diff
@@ -122,6 +122,8 @@ namespace FileUtils {
     // it contains at least one file other than "." or "..".
     bool isDirectoryNotEmpty(const tstring &dirPath);
 
+    tstring_array listContents(const tstring& basedir, const tstring& filename);
```

Comment:
It would be better to provide implementation of `void iterateDirectory(const tstring &dirPath, DirectoryCallback& callback)` from WinFileUtils.h on UNIX platforms. This would allow the reusing of code that relies on it from WinFileUtils on other platforms.